src/main/java/com/sweng894/GetVaccinated/api/config/DynamoDbConfiguration.java

```diff
@@ -1,43 +1,42 @@
 package com.sweng894.GetVaccinated.api.config;
 
-import org.springframework.context.annotation.Bean;
-import org.springframework.context.annotation.Configuration;
-import org.springframework.context.annotation.Bean;
-import org.springframework.context.annotation.Configuration;
 import com.amazonaws.auth.AWSStaticCredentialsProvider;
 import com.amazonaws.auth.BasicAWSCredentials;
 import com.amazonaws.client.builder.AwsClientBuilder;
 import com.amazonaws.services.dynamodbv2.AmazonDynamoDB;
 import com.amazonaws.services.dynamodbv2.AmazonDynamoDBClientBuilder;
 import com.amazonaws.services.dynamodbv2.datamodeling.DynamoDBMapper;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
 
 
 @Configuration
 public class DynamoDbConfiguration {
     @Bean
-    public DynamoDBMapper DynamoDBMapper() {
-        return new DynamoDBMapper(buildAmazonDynamoDb());
+    public DynamoDBMapper dynamoDBMapper() {
+      return new DynamoDBMapper(amazonDynamoDB());
+    }
+    @Bean
+    public AmazonDynamoDB amazonDynamoDB() {
+      var accessKeyId = System.getenv("AWS_ACCESS_KEY_ID");
+      var awsSecretAccess = System.getenv("AWS_SECRET_ACCESS_KEY");
+      var region = System.getenv("AWS_DEFAULT_REGION");
```

Comment:
It will be possible to use a normal non-local DynamoDB by specifying the AWS_* environment variables.